src/ipa/rpi/common/ipa_base.cpp

```diff
@@ -155,6 +155,7 @@ int32_t IpaBase::init(const IPASettings &settings, const InitParams &params, Ini
 	lensPresent_ = params.lensPresent;
 
 	controller_.initialise();
+	helper_->setHwConfig(controller_.getHardwareConfig());
```

Comment:
Ah, so being "strided" is to do with parsing the sensor embedded data related to the CNN?

Reply:
Correct - this is a difference how the data buffers are strided between Unicam and the CFE.  We get away with this in the IMX708 and other embedded data parsing code since we don't explicitly look through lines.